src/novo/src/server.ts

```diff
@@ -86,51 +101,36 @@ function initializeNovo() {
         const headTagsString = ReactDOM.renderToString(headTags as any)
         const sharifyData = res.locals.sharify.script()
 
-        res.status(status).send(`
-          <html>
-            <head>
-              ${styleTags}
-              ${headTagsString}
-              ${sharifyData}
-            </head>
-            <body>
-              <script src="${novoManifest.lookup(
-                "/assets-novo/novo-runtime.js"
-              )}"></script>
-              <script src="${novoManifest.lookup(
-                "/assets-novo/novo-common.js"
-              )}"></script>
-              <script src="${novoManifest.lookup(
-                "/assets-novo/novo-artsy-common.js"
-              )}"></script>
-              <script src="${novoManifest.lookup(
-                "/assets-novo/novo-common-react.js"
-              )}"></script>
-              <script src="${novoManifest.lookup(
-                "/assets-novo/novo-common-utility.js"
-              )}"></script>
-              <script src="${novoManifest.lookup(
-                "/assets-novo/novo-artsy.js"
-              )}"></script>
-
-              <div id='react-root'>
-              ${bodyHTML}
-              </div>
-
-              ${scripts}
-              <script src="${novoManifest.lookup(
-                "/assets-novo/novo-artsy-novo.js"
-              )}"></script>
-            </body>
-          </html>
-        `)
+        const options = {
+          content: {
+            body: bodyHTML,
+            data: sharifyData,
+            head: headTagsString,
+            scripts,
```

Comment:
If we're using HTML Webpack Plugin now  🎉🎉🎉 at some future point it might be worth checking out the react-loadable docs to see if there's some way we can make the script aggregation process simpler; I _think_ there's some support for the plugin